src/pages/progress_reports/CloseoutReview.tsx

```diff
@@ -1,113 +1,176 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
 import InfoIcon from '@mui/icons-material/Info';
+import {Acknowledgement} from 'src/types/model';
+import {User} from 'src/libs/ajax/User';
+import {Notifications} from 'src/libs/utils';
+import {extractConsentError, extractError} from 'src/utils/ErrorUtils';
 
 interface CloseoutReviewProps {
-    onApprove?: () => void;
-    onReturn?: () => void;
+  onApprove?: () => void;
+  onReturn?: () => void;
+  referenceId?: string;
 }
 
 export const CloseoutReview: React.FC<CloseoutReviewProps> = ({
+    referenceId,
     onApprove,
     onReturn,
 }) => {
-    return (
-        <div className="progress-report-step-card" style={{
-            border: '1px solid #4D72AA',
+
+  const [acknowledgement, setAcknowledgement] = useState<Acknowledgement | undefined>(undefined);
+
+  // Required to get Chairperson acknowledgments of closeouts
+  useEffect(() => {
+    // Fetch the acknowledgement for the given referenceId
+    const fetchAcknowledgement = async () => {
+      if (referenceId) {
```

Comment:
Does it make sense to use a promise pattern here?

Have getAcknowledgement return a promise-like response and handle the catch with a standard error object?

pseudocode:
User.getAcknowledgement(myKey).then(response => {
    setAcknowledgement(response.data)
}).catch( error => {
if (error.response && error.response.status != 404) {
Notifications.showError({...fill in the error to display})
}
});